apps/web/src/components/EmployeeSearchInput/Error.tsx

```diff
@@ -90,6 +90,20 @@ const useDefaultMessages = (email: string | undefined): ErrorMessages => {
           "Description of default message when an email is not a government of Canada email",
       }),
     },
+    NOT_VERIFIED_GOVERNMENT_EMPLOYEE: {
+      title: intl.formatMessage({
+        defaultMessage: "This user hasn't verified their employee status",
+        id: "zVGK45",
+        description: "Label for when an employee was found but not verified",
+      }),
+      body: intl.formatMessage({
+        defaultMessage:
+          "We found a user with the email address provided, but they haven’t confirmed their status as an employee. Only verified employees can be nominated at this time. The user can verify their status as an employee by logging into the platform and verifying their work email.",
+        id: "hYuDZ2",
+        description:
+          "Descrioption of default message when user was found but not a verified employee",
```

Comment:
```suggestion
          "Description of default message when user was found but not a verified employee",
```

Reply:
Thanks, fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13363/commits/ccde35c6870674db326d9a2046999c241713edd2)